Makefile

```diff
@@ -17,8 +17,12 @@
 
 GO ?= go
 POSTGRES_TEST_TAG ?= 20170308-1644
-EXAMPLES_ORMS_PATH = /examples-orms
-DOCKER = docker run --volume="$(shell pwd)":$(EXAMPLES_ORMS_PATH) docker.io/cockroachdb/postgres-test:$(POSTGRES_TEST_TAG)
+EXAMPLES_ORMS_PATH = /root/go
```

Comment:
I'd make this:

```make
DOCKER_GOPATH := /root/go
DOCKER_REPO_PATH := $(DOCKER_GOPATH)/src/github.com/cockroachdb/examples-orms
```

and then use `DOCKER_GOPATH` and `DOCKER_REPO_PATH` appropriately below.